docs/product-assembly_running_employee_rostering_in_wb/src/main/asciidoc/wb-employee-rostering-shift-required-skills-met-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='wb-employee-rostering-shift-required-skills-met-proc']
+= Create the `ShiftReqiredSkillsAreMet` guided rule 
+
+The `ShiftReqiredSkillsAreMet` guided rule creates a hard constraint that ensures all shifts are assigned an employee with the correct set of skills. In the employee rostering example, this constraint is created using the guided rule editor. 
+
+image::ShiftRequiredSkillsAreMet.png[ShiftRequiredSkillsAreMet Guided Rule]
+
+.Procedure
+. Go to *Menu* -> *Design* -> *Projects* and click the project name.
+. Click *Create New Asset* -> *Guided Rule*.
+. Enter `ShiftReqiredSkillsAreMet` as the *Guided Rule* name and select the `employeerostering.employeerostering` *Package*. 
+. Click *Ok* to create the rule asset.
+. Add a *WHEN* condition by clicking the image:5686.png[] in the *WHEN* field.
+. Select `ShiftAssignment` in the *Add a condition to the rule...* window. Click *+Ok*. 
+. Click on the `ShiftAssignment` condition, and select `employee` from the *Add a restriction on a field* drop down list.
```

Comment:
Should typically avoid single numbered steps. Just make part of the actual step instead a single sub-step?